.gitignore

```diff
@@ -1,3 +1,2 @@
 .idea/
 *.gem
-Gemfile.lock
```

Comment:
That generally best practice to keep lock file ignored as that is a gem, not an app.